hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.cc

```diff
@@ -20,6 +20,11 @@ namespace wireless_android_play_analytics{
 
 void HologramDataSourceAvailabilityFetcher::Process() {
     // TODO(alexanderlin): add implementation.
+    assert(FLAGS_config_file_path is not empty)
+    // AcquireConfig(FLAGS_config_file_path)
+    // FetchFromDatabase()
+    // GetStatus(todays date)
```

Comment:
Where does the date come from?

You need to define a flag to pass in the date.

Reply:
I was thinking you could simply get today's date with the civil date library

Reply:
That's not a good practice to use things like NowTime() to get CivilDay in the job. We want to keep the system idempotent between different runs. A common practice is passing a Flag that takes date into the system.

Reply:
got it will add a flag